spec/controllers/admin/clipboard_controller_spec.rb

```diff
@@ -18,13 +18,13 @@ module Alchemy
 
       describe "#insert" do
         it "should hold element ids" do
-          xhr :post, :insert, {remarkable_type: 'elements', remarkable_id: element.id}
+          alchemy_xhr :post, :insert, {remarkable_type: 'elements', remarkable_id: element.id}
```

Comment:
Line is too long. [94/80]<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.